packages/devextreme/js/ui/chat.js

```diff
@@ -1,3 +1,40 @@
 import Chat from '../__internal/ui/chat/chat';
 
 export default Chat;
+
+/**
+ * @name dxChatOptions.accessKey
+ * @hidden
+ */
+
+/**
+ * @name dxChatOptions.activeStateEnabled
+ * @hidden
+ */
+
+/**
+ * @name dxChatOptions.focusStateEnabled
+ * @hidden
+ */
+
+/**
+ * @name dxChatOptions.hint
+ * @hidden
+ */
+
+/**
+ * @name dxChatOptions.onContentReady
+ * @hidden true
+ * @action
+ */
+
+/**
+ * @name dxChatOptions.tabIndex
+ * @hidden
+ */
+
+/**
+ * @name dxChat.focus
```

Comment:
Why we decide to hide focus method? It might be used for focusing of the field.

Reply:
[4c429a74d08cebbf0cb75a28fa23ed6d511559ba]